### PR TITLE
cookiecutter helm

### PR DESCRIPTION
**Pull request by lachtanek**
Also fixes issue when app_name contains a dash (python needs underscore)

### CODE REVIEW

File: {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/templates/hpa.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{- if .Values.autoscaling.enabled }}
+apiVersion: autoscaling/v2beta1
+kind: HorizontalPodAutoscaler
```
Comment (jerguslejko) [inline {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/templates/hpa.yaml:3]: do we need hpa for all services? I don't think we're using it for any existing ones

File: {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/values.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+service:
+  enabled: true
+  type: ClusterIP
+  port: 80
+ingress:
+  enabled: false
+  annotations: {}
+  tls: []
+  hosts:
+    - host: chart-example.local
```
Comment (jerguslejko) [inline {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/values.yaml:25]: this should change probably

**Review by jerguslejko — APPROVED** [review]

File: {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/templates/hpa.yaml (hunk 1 above)
Comment (lachtanek) [reply {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/templates/hpa.yaml:3]: we are now for document-generator

File: {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/values.yaml (hunk 2 above)
Comment (lachtanek) [reply {{ cookiecutter.app_name }}/charts/{{ cookiecutter.app_name }}/values.yaml:25]: if anyone specifies `enabled: true` in values, they should override hosts also, not sure what the best practice is, for showing the structure, but I'd leave it as-is for now

**Pull request merged**